indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -127,7 +118,7 @@ def _process(self, args):
         if not self._validate_room_booking_limit(args['start_dt'], args['end_dt'], booking_limit_days):
             msg = (_('Bookings for the room "{}" may not be longer than {} days')
                    .format(room.name, booking_limit_days))
-            return jsonify(success=False, msg=msg)
+            raise ExpectedError(message=msg)
```

Comment:
`raise ExpectedError(msg)` (it's pretty uncommon to use a kwarg for an exception's message)